README.md

```diff
@@ -315,7 +315,7 @@ The address of the Emotable repository smart contract is designed to resemble th
 Designing the proposal, we considered the following questions:
 
 1. **Does the proposal support custom emotes or only the Unicode specified ones?**\
-The proposal only accepts the Unicode identifier which is a `bytes4` value. This means that while we encourage implementers to add the reactions using standardized emojis, the values not covered by the Unicode standard can be used for custom emotes. The only drawback being that the interface displaying the reactions will have to know what kind of image to render and such additions will probably be limited to the interface or marketplace in which they were made.
+The proposal only accepts the Unicode identifier which is a `string` value. This means that while we encourage implementers to add the reactions using standardized emojis, the values not covered by the Unicode standard can be used for custom emotes. The only drawback being that the interface displaying the reactions will have to know what kind of image to render and such additions will probably be limited to the interface or marketplace in which they were made.
```

Comment:
I think we should also address the initial plan to use `bytes4` and why we updated it to `string` type 🤔

Reply:
Totally. I was going to ask you. Can you take care? Basically, the current emojis have an arbitrary length so we would have to use way bigger bytes array at which point it's no longer an optimization against strings. Also strings make a much more usable dev experience